### PR TITLE
python3Packages.litemapy: init at 0.7.2b0, python3Packages.nbtlib: init at 2.0.4

### PR DESCRIPTION
**Pull request by gabriel-doriath-dohler**
###### Description of changes

litemapy: read and edit Litematica's schematic file format in python (https://pypi.org/project/litemapy/)
nbtlib: read and edit [nbt data](http://wiki.vg/NBT) (https://pypi.org/project/nbtlib/)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/litemapy/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  ];
+
+  pythonImportsCheck = [ "litemapy" ];
+
+  meta = with lib; {
+    description = "Litemapy's goal is to provide an easy to use way to read and edit Litematica's schematic file format";
+    homepage = "https://github.com/SmylerMC/litemapy";
+    changelog = "https://github.com/SmylerMC/litemapy/blob/${src.rev}/CHANGELOG.md";
+    license = licenses.gpl3Only;
+    maintainers = with maintainers; [ ];
```
Comment (RaitoBezarius) [inline pkgs/development/python-modules/litemapy/default.nix:30]: missing maintainer here :p

File: pkgs/development/python-modules/litemapy/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  };
+
+  propagatedBuildInputs = [
+    nbtlib
+  ];
+
+  pythonImportsCheck = [ "litemapy" ];
+
+  meta = with lib; {
+    description = "Litemapy's goal is to provide an easy to use way to read and edit Litematica's schematic file format";
```
Comment (RaitoBezarius) [inline pkgs/development/python-modules/litemapy/default.nix:26]:
```suggestion
    description = "A Python library to read and edit Litematica's schematic file format";
```

**Comment by JulienMalka** [comment]
Result of `nixpkgs-review pr 240742` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built:</summary>
  <ul>
    <li>python310Packages.litemapy</li>
    <li>python310Packages.litemapy.dist</li>
    <li>python310Packages.nbtlib</li>
    <li>python310Packages.nbtlib.dist</li>
    <li>python311Packages.litemapy</li>
    <li>python311Packages.litemapy.dist</li>
    <li>python311Packages.nbtlib</li>
    <li>python311Packages.nbtlib.dist</li>
  </ul>
</details>

**Review by JulienMalka — APPROVED** [review]
LGTM :) Thanks for your first contribution !

**Review by RaitoBezarius — APPROVED** [review]

**Pull request merged**